### ISSUE TITLE
Content within the notification banner uses the default user-agent text colour

### ISSUE THREAD

**Issue by 36degrees** [alphagov/govuk-frontend#2037]
## Description of the issue

The notification banner content currently uses the user agent default text colour.

This means that for users who have not changed their user agent default text colour, the text will be true black (`#000000`) rather than the ‘GOV.UK’ black we specify in the colour palette (`#0b0c0c`).

It also means that if a user has changed their default text colour, but has not chosen to override colours, depending on the user’s prefered colour scheme the text may have low contrast or be invisible.

As I understand it, this is as a [‘failure of Success Criterion 1.4.3, 1.4.6 and 1.4.8 due to specifying foreground colors without specifying background colors or vice versa’][1].

## Steps to reproduce the issue

For example, my Firefox instance is configured to use white text on a blue background:

![Firefox colour preferences, with 'Text' set to white, 'Background' set to blue, and Unvisited/Visited links set to yellow, and 'Override the colors specified by the page with your selections above' set to 'Never'](https://user-images.githubusercontent.com/121939/100221111-231cbc00-2f10-11eb-9f44-f9ff0009031b.png)

## Actual vs expected behaviour

The notification banner is therefore white-on-white, and thus invisible:

![The notification banner showing invisible text](https://user-images.githubusercontent.com/121939/100221138-2a43ca00-2f10-11eb-81bf-6c563a49ea2b.png)

Rather than:

![The fixed notification banner with visible text](https://user-images.githubusercontent.com/121939/100221274-5a8b6880-2f10-11eb-815c-d04e6d31c6da.png)

[1]: https://www.w3.org/TR/WCAG20-TECHS/F24.html

**Comment by 36degrees** [comment]
This is tied into https://github.com/alphagov/govuk-frontend/issues/1772 – if a user is applying `govuk-template__body` to the `<body>` class then we set a background colour but not a foreground colour, so this is really that issue propagating down to the component.

If the user is not using our `govuk-template__body` then there's a reasonable chance that they're setting both the background and foreground colours on the `<body>` in which case this issue would not affect them.
